src/content/self-hosted/administration/configuration.mdx

```diff
@@ -517,6 +517,23 @@ The role Okteto assigns to every user. If empty, Okteto will create a default ro
 clusterRole: "role name"
 ```
 
+### extraRoleBindings
+
+Extra `RoleBindings` to be created for every user's service account. This is useful if you want to grant access to other namespace resources in the cluster.  Disabled by default.
+
+The `RoleBindings` will reference `ClusterRoles` and will be scoped to the namespace specified in the configuration. Here's an example:
+
+```yaml
+extraRoleBindings:
```

Comment:
@rberrelleza maybe we should move this under `users`? as we did for `serviceAccounts`?

Reply:
+1. I was just going to post that. Since we are adding these as new features, it'd be nice to have them organized.

Reply:
Yeah, I agree